### PR TITLE
Add BOB Network

### PR DESCRIPTION
**Pull request by leonimella**

### CODE REVIEW

File: sdks/sdk-core/src/addresses.ts [hunk 1]
```diff
@@ -271,6 +272,16 @@ const ZERO_ADDRESSES: ChainAddresses = {
   swapRouter02Address: '0xD936711eABD2Ce52747d7122757316C7DFe3599b',
 }
 
+const BOB_ADDRESSES: ChainAddresses = {
+  v3CoreFactoryAddress: '0xcb2436774C3e191c85056d248EF4260ce5f27A9D',
+  multicallAddress: '0x5d6b0f5335ec95cD2aB7E52f2A0750dd86502435',
+  quoterAddress: '0x6Aa54a43d7eEF5b239a18eed3Af4877f46522BCA',
+  // v3MigratorAddress: 'NA',
```
Comment (leonimella) [inline sdks/sdk-core/src/addresses.ts]: Hey @msvstj, This contract was not filled in the spreadsheet so I comment it out.

File: sdks/sdk-core/src/entities/weth9.ts [hunk 2]
```diff
@@ -26,4 +26,5 @@ export const WETH9: { [chainId: number]: Token } = {
   324: new Token(324, '0x5AEa5775959fBC2557Cc8789bC1bf90A239D9a91', 18, 'WETH', 'Wrapped Ether'),
   11124: new Token(11124, '0x9EDCde0257F2386Ce177C3a7FCdd97787F0D841d', 18, 'WETH', 'Wrapped Ether'),
   543210: new Token(543210, '0xAc98B49576B1C892ba6BFae08fE1BB0d80Cf599c', 18, 'WETH', 'Wrapped Ether'),
+  60808: new Token(543210, '0x4200000000000000000000000000000000000006', 18, 'WETH', 'Wrapped Ether'),
```
Comment (leonimella) [inline sdks/sdk-core/src/entities/weth9.ts:29]: I notice this file, and did a quick search for the BOB's wETH contract, but I'm not sure if this is the correct one for the network.

File: sdks/universal-router-sdk/src/utils/constants.ts [hunk 3]
```diff
@@ -294,6 +294,19 @@ export const CHAIN_CONFIGS: { [key: number]: ChainConfig } = {
       },
     },
   },
+  [60808]: {
+    weth: '0x4200000000000000000000000000000000000006',
+    routerConfigs: {
+      [UniversalRouterVersion.V1_2]: {
+        address: 'NA',
```
Comment (leonimella) [inline sdks/universal-router-sdk/src/utils/constants.ts]: These addresses were also missing from the spreadsheet

File: sdks/sdk-core/src/entities/weth9.ts (hunk 2 above)
Comment (msvstj) [reply sdks/sdk-core/src/entities/weth9.ts:29]: It is a correct one. https://explorer.gobob.xyz/address/0x4200000000000000000000000000000000000006

File: sdks/universal-router-sdk/src/utils/constants.ts (hunk 3 above)
Comment (msvstj) [reply sdks/universal-router-sdk/src/utils/constants.ts]: Universal router is this address: 0x346239972d1fa486FC4a521031BC81bFB7D6e8a4 
https://explorer.gobob.xyz/address/0x346239972d1fa486FC4a521031BC81bFB7D6e8a4 (for blocknumber)

File: sdks/v3-sdk/src/constants.ts [hunk 4]
```diff
@@ -13,6 +13,8 @@ export function poolInitCodeHash(chainId?: ChainId): string {
     case ChainId.ABSTRACT_TESTNET:
     case ChainId.ZERO:
       return '0x010013f177ea1fcbc4520f9a3ca7cd2d1d77959e05aa66484027cb38e712aeed'
+    case ChainId.BOB:
+      return '0x96e8ac4277198ff8b6f785478aa9a39f403cb768dd02cbee326c3e7da348845f'
     default:
       return POOL_INIT_CODE_HASH
```
Comment (msvstj) [inline sdks/v3-sdk/src/constants.ts]: I'd revert changes for BOB since POOL_INIT_CODE_HASH already has a default value for BOB

**Pull request merged**